app/src/test/java/bubi/com/logintest/TestRunner.java

```diff
@@ -12,20 +12,20 @@
     private DetailsScreenTest detailsScreenTest = new DetailsScreenTest();
     private SomeOtherScreenTest someOtherScreenTest = new SomeOtherScreenTest();
 
-    @Test
-    public void runHomeScreenTest(){
-        homeScreenTest.run();
-    }
-
-    @Test
-    public void runDetailsScreenTest(){
-        detailsScreenTest.run();
-
-    }
-
-    @Test
-    public void runSomeOtherScreenTest(){
-        someOtherScreenTest.run();
-    }
+//    @Test
```

Comment:
@silviadimova please remove the code, do not leave comments in the code

Reply:
OK, thanks